### PR TITLE
[AIRFLOW-2031] Add missing param in the eg in DataFlow docstrings

### PR DESCRIPTION
**Pull request by kaxil**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "[AIRFLOW-XXX] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2031

### Description
- [x] Here are some details about my PR, including screenshots of any UI changes:
Added `gcp_conn_id` parameter in the examples provided in docstrings for DataFlow operators.

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Just addition to docstrings, hence no tests required

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by kaxil** [comment]
@criccomini PTAL

File: airflow/contrib/operators/dataflow_operator.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ class DataFlowJavaOperator(BaseOperator):
             'partitionType': 'DAY',
             'labels': {'foo' : 'bar'}
         },
+        gcp_conn_id='google_cloud_default',
```
Comment (fenglu-db) [inline airflow/contrib/operators/dataflow_operator.py]: mind replacing 'google_cloud_default' with another connection name? 'google_cloud_default' is the default value for gcp_conn_id. Here and the other place.

**Comment by kaxil** [comment]
Done @fenglu-g

**Review by fenglu-db — COMMENTED** [review]
LGTM, thanks! 

@criccomini, kindly help to merge.

**Comment by Fokko** [comment]
Merged, thanks all!

**Pull request closed without merging**